source/CardGames.py

```diff
@@ -432,6 +433,74 @@ def marshall_first_turn(self):
       # Print the board at the end of turn
       self.display_board_general()
 
+    def FugitiveRepeating(self):
+      draw = 0
+      NewHideouts = []
+      while draw != 1 and draw != 2 and draw != 3:
+        try:
+          draw = int(input("Select which deck to draw a card from ['1' (1-14), '2' (15-28), '3' (29-42)]: ").strip())
+        except:
+          ValueError
+      if draw == 1:
+        self.fugitive_deck.append(self.LowRangeDeck.pop())
+      elif draw == 2:
+        self.fugitive_deck.append(self.MidRangeDeck.pop())
+      elif draw == 3:
+        self.fugitive_deck.append(self.HighRangeDeck.pop())
+
+      card_value = self.fugitive_deck[-1].value
+      sprint_value = self.fugitive_deck[-1].sprint_value
+      print("You drew " + str(card_value) + " with a sprint value of " + str(sprint_value) + ".")
+
+      turn = ""
+      while turn != "hideout" and turn != "pass":
+        turn = input("Select if you want to place a hideout(s) or pass ['hideout' for hideout or 'pass' for pass]: ").strip().lower()
+      if turn == "hideout":
+        check = 2
+        while check != 0:
+          try:
+            backup = NewHideouts[:]
+            check = int(input("If you no longer want to place hideouts, type 0 otherwise type 1: ").strip())
+          except:
+            ValueError
+            continue
+          good = True
+          if check == 1:
+            good = False
+          while good == False:
+              bad = False
+              NewHideouts += [input("Select which card you want to place as a hideout followed by the cards you want to burn separated only by a comma (1,2,3...): ").strip().split(',')]
+              for y in NewHideouts:
+                for z in y:
+                  try:
+                      if int(z) > 42 or int(z) < 1:
+                        print("Please input a valid hideout.")
+                        bad = True
+                        good = False
+                        break
+                  except:
+                    ValueError
+                    print("Please input a valid hideout.")
+                    bad = True
+                    good = False
+                    NewHideouts = backup   
+                    break      
+                  if len(z) > 1:
+                    for i in z:
+                      if i not in "123456789101112131415161718192021222324252627282930313233343536373839404142":
+                        print("Please input a valid hideout.")
+                        bad = True
+                        good = False
+                        break
+                if bad == True:
+                  NewHideouts = backup
+                  break
+              good = True
+              #Functions to check for illegally placed hideouts and burns would go here
+      elif turn == "pass":
+        print("Passing...")
+      return NewHideouts
+
```

Comment:
Looks great brother. Are you able to comment out the brun functionality for now so we can focus on the main functionaility?

Reply:
Thanks, Georgio! I will be sure to comment out the burn functionality for now.